package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "restful-react",
-  "version": "14.2.1",
+  "version": "14.2.2",
```

Comment:
Since we add a new feature, don't be shy, you can bump the minor number 😉

Reply:
thx, I had the same feeling but wasn't sure) Fixed